packages/generator/generators/react-component/index.js

```diff
@@ -106,5 +130,49 @@ module.exports = yeoman.Base.extend({
 				this
 			);
 		}
+		if (this.props.parentIndex) {
+			const indexPath = path.join(this.props.path, './index.js');
+			let code;
+			try {
+				code = fs.readFileSync(indexPath);
+			} catch (error) {
+				this.log(error.message);
+			}
+			if (code) {
+				const ast = recast.parse(code);
+				const b = recast.types.builders;
+				// first find last import
+				const imports = ast.program.body.filter(item => item.type === 'ImportDeclaration');
+				const lastIndexImport = ast.program.body.indexOf(imports[imports.length - 1]);
```

Comment:
please don't make computation inside function call, extract the `lastImport` in a variable